dom.bs

```diff
@@ -4097,14 +4085,9 @@ A <a>node</a> <var>A</var>
    <dt>Any other node
    <dd>&mdash;
   </dl>
- <li>If <var>A</var> is an <a for="/">element</a>, each
- <a>attribute</a> in its
- <a for=Element>attribute list</a> has an
- <a>attribute</a> with the same
- <a for=Attr>namespace</a>,
- <a for=Attr>local name</a>, and
- <a for=Attr>value</a> in <var>B</var>'s
- <a for=Element>attribute list</a>.
+ <li>If <var>A</var> is an <a for="/">element</a>, each <a>attribute</a> in its
```

Comment:
Oh, I missed that up in "Its namespace, namespace prefix, local name, and its number of attributes in its attribute list." Never mind then :)